Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/interoperability/InteroperabilityConfigurationFragmentViewModelTest.kt

```diff
@@ -0,0 +1,71 @@
+package de.rki.coronawarnapp.ui.interoperability
+
+import androidx.lifecycle.MutableLiveData
+import de.rki.coronawarnapp.storage.interoperability.InteroperabilityRepository
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.verify
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import testhelpers.extensions.InstantExecutorExtension
+import testhelpers.extensions.getOrAwaitValue
+
+@Extensions(
+    ExtendWith(MockKExtension::class),
+    ExtendWith(InstantExecutorExtension::class)
+)
+class InteroperabilityConfigurationFragmentViewModelTest {
+
+    @MockK(relaxUnitFun = true)
+    lateinit var interoperabilityRepository: InteroperabilityRepository
+
+    private lateinit var interoperabilityConfigurationFragmentViewModel: InteroperabilityConfigurationFragmentViewModel
+
+    private val countryListLiveData = MutableLiveData<List<String>>(listOf())
+
+    @BeforeEach
+    fun setupFreshViewModel() {
+        every { interoperabilityRepository.countryList } returns countryListLiveData
+        interoperabilityConfigurationFragmentViewModel =
+            InteroperabilityConfigurationFragmentViewModel(interoperabilityRepository)
+    }
+
+    @Test
+    fun testNoFetch() {
```

Comment:
IMHO the "test" prefix is to be dropped as all functions are tests - no added value.
Moreover, the name of the text could describe in more detail what is being tested. E.g. "countryListIsEmptyIfRepositoryReturnsNoData" or alike